index.html

```diff
@@ -733,195 +747,213 @@
+]
 ]
 ---
 .left-column[
-  ## Presenter mode
+  ## Powitanie
+  ## Mapa
+  ## Widok podstawowy
 ]
 .right-column[
-To help out with giving presentations, a presenter mode comprising the
-following features is provided:
 
-- Display of slide notes for the current slide, to help you remember
-  key points
+.text-content[
 
-- Display of upcoming slide, to let you know what's coming
+Prezentacja danych dostarczonych z serwera:
+
+- czytelna i prosta prezentacja podstawowych danych
+
+- reaktywne ładowanie poszczególnych elemntów
+
+- podgląd do aktualnego zapełnienia miejsca
 
-- Cloning of slideshow for viewing on extended display
+]
+
+.image-content[
+<img src="./img/details_light.jpg" alt="details" height="400px"/>
+]
 ]
 ---
 .left-column[
-  ## Presenter mode
-  ### - Inline notes
+  ## Powitanie
+  ## Mapa
+  ## Widok podstawowy
+  ## Statystyki
 ]
 .right-column[
-Just like three dashes separate slides,
-three question marks separate slide content from slide notes:
 
-```
-Slide 1 content
+.text-content[
 
-*???
+Przygotowanie narzędzi do prezentacji statystyk:
 
-Slide 1 notes
+- gotowe wykresy słupkowe oraz liniowe
+
+- powiązanie danych z wykresami (**WIP**)
+
+- możliwość zmiany prezentacji danych (**WIP**)
+
+]
 
+.image-content[
+<img src="./img/stats_light.jpg" alt="stats" height="400px"/>
+]
+]
 ---
+.left-column[
+  ## Powitanie
+  ## Mapa
+  ## Widok podstawowy
+  ## Statystyki
+  ## Menu
+]
+.right-column[
 
-Slide 2 content
+.text-content[
 
-*???
+Prezentacja stałego menu danego miejsca:
 
-Slide 2 notes
-```
+- nazwy, ceny, opinie o produktach
 
-Slide notes are also treated as Markdown, and will be converted in the
-same manner slide content is.
+- możliwość wystawienia własnej oceny jedzenia (**WIP**)
 
-Pressing __P__ will toggle presenter mode.
 ]
-???
-Congratulations, you just toggled presenter mode!
 
-Now press __P__ to toggle it back off.
+.image-content[
+<img src="./img/food_light.jpg" alt="food" height="400px"/>
+]
+]
 ---
 .left-column[
-  ## Presenter mode
-  ### - Inline notes
-  ### - Cloned view
+  ## Powitanie
+  ## Mapa
+  ## Widok podstawowy
+  ## Statystyki
+  ## Menu
+  ## Ustawienia
 ]
 .right-column[
-Presenter mode of course makes no sense to the audience.
 
-Creating a cloned view of your slideshow lets you:
+.text-content[
 
-- Move the cloned view to the extended display visible to the audience
+Możliwość zmiany preferencji aplikacji w perzystentny sposób:
 
-- Put the original slideshow in presenter mode
+- dwa jęzki obsługi aplikacji - polski i angielski
 
-- Navigate as usual, and the cloned view will automatically keep up with
- the original
+- dwa motywy obsługi aplikacji - podstawowy i ciemny
 
-Pressing __C__ will open a cloned view of the current slideshow in a new
-browser window.
-]
----
-template: inverse
+- aktualna wersja aplikacji jako wejście do *dev mode* (**WIP**)
 
-## A niedługo czas, aby każdy mógł spróbować 
+- nickname do opinii (**WIP**)
```

Comment:
co to znaczy

Reply:
ze ebdzie mozliwosc ustawienia sobie username do wystawiania opinii do jedzenia, napisze ladniej :)